src/Datadog.Trace/Tracer.cs

```diff
@@ -348,6 +349,40 @@ private static IStatsd CreateDogStatsdClient(TracerSettings settings, string ser
             return new Statsd(statsdUdp, new RandomGenerator(), new StopWatchFactory(), prefix: string.Empty, constantTags);
         }
 
+        private static void StartTraceAgentWhenNecessary()
```

Comment:
Thoughts on making this not AAS-specific and instead just look for an agnostic env var like `DD_TRACE_AGENT_PATH`? If it's not define, so nothing. If it's defined, and the file exists, and a process with the same name does not exist, then try to start it.

Reply:
That's fine, I can make that change. Other option is checking to see if the host/port is bound already, because checking for the name won't be as safe as that IMO. Thoughts?

Reply:
I have a slight preference for Lucas's suggestion, but both work to achieve the good suggestion of not limiting this to AAS.

Reply:
I went with name only for now, and named the trace agent reasonably uniquely. 
We can expand it later as we find the need to.